### PR TITLE
add PhpStorm plugin on third party integration

### PR DESCRIPTION
**Pull request by agallou**
before : 
capture d ecran 2016-02-01 a 19 49 24

after :
capture d ecran 2016-02-01 a 19 50 03

### CODE REVIEW

File: features.md [hunk 1]
```diff
@@ -276,7 +276,12 @@ atoum integrates well with those CI tools:
 </ul>
 
 atoum has a **great** integration with those IDEs:
-<ul class="columns integrations" data-columns="3">
+<ul class="columns integrations" data-columns="4">
+    <li>
+      <img src="images/integrations_logos/phpstorm.svg" />
+      <a href="https://atom.io/">PhpStorm</a>
```
Comment (jubianchi) [inline features.md]: the link is wrong here

**Comment by jubianchi** [comment]
only a tiny typo left before we merge this ;)

:+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1:

**Comment by agallou** [comment]
@jubianchi typo fixed

**Comment by Hywan** [comment]
Nice :-).

**Pull request merged**